webpack/ForemanTasks/Components/TasksTable/TasksTableSelectors.js

```diff
@@ -0,0 +1,43 @@
+import { selectForemanTasks } from '../../ForemanTasksSelectors';
+
+export const selectTasksTable = state =>
+  selectForemanTasks(state).tasksTable || {};
+
+export const selectTasksTableContent = state =>
+  selectTasksTable(state).tasksTableContent || {};
+
+export const selectTasksTableQuery = state =>
+  selectTasksTable(state).tasksTableQuery || {};
+
+export const selectQuery = state => selectTasksTableQuery(state).query || {};
+
+export const selectLoading = state =>
+  selectTasksTableQuery(state).loading || false;
+
+export const selectPagitation = state =>
+  selectTasksTableQuery(state).pagination || {};
+
+export const selectItemCount = state =>
+  selectTasksTableQuery(state).itemCount || 0;
+
+const formatDate = date =>
+  date
+    ? new Date(date).toLocaleString('en-GB', {
```

Comment:
Hardcoded locale? Shouldn't it be retrieved from the environment or something?

Reply:
I didnt want there to be a confusion with the search text as the date for 'en-US' would be MM/DD/YYYY in the table, but it can't be searched in this format